### PR TITLE
Added Verify Rate Limit Info

### PR DESCRIPTION
**Pull request by christinechoy-okta**
## Description:
- **What's changed?** Added verification info
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-428184](https://oktainc.atlassian.net/browse/OKTA-428184)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md [hunk 1]
```diff
@@ -96,16 +96,7 @@ API endpoints that take username and password credentials, including the [Authen
...
-
-  **Endpoints**
-  * `/api/v1/authn/factors`
-  * `/api/v1/users/${userId}/factors`
-
-  **Identity Engine endpoints**<br>
-  <ApiLifecycle access="ie" />
-  * `/idp/idx/challenge`
-  * `/idp/idx/credential/enroll`
+* **Enrollment verification and rate limits:** This rate limit applies to a user's attempt to enroll or verify an [SMS or a Call factor](/docs/reference/api/factors/) using any phone number. This rate limit applies to only the enrollment or verification operation. See [System Log events for rate limits](/docs/reference/rl-system-log-events/#debugcontext-object-examples) for examples of System Log rate limit events where too many enrollment or verification attempts for the SMS or Call factors were made.
```
Comment (scottbermon-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md:99]: I think this heading should be:
```
**Enrollment and verification rate limits:**
```

Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md]: This anchor doesn't exist? #debugcontext-object-examples
Maybe #debugcontext-object-examples-for-operation-rate-limits ??

**Review by vanngo-okta — APPROVED** [review]
Just a question about the link. Otherwise, looks good!

**Review by scottbermon-okta — APPROVED** [review]
LGTM

File: packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md (hunk 1 above)
Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md]: Anchor fixed.

Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md]: Fixed.

**Pull request merged**